### ISSUE TITLE
Redirects and other issues in /sell/orders/orders/new when adding address, customer, showing order/product details etc

### ISSUE THREAD

**Issue by madsoliver** [PrestaShop/PrestaShop#17223]
When I add a new order using the new controller i constantly get kicked out when doing the following tasks:

1. ✅ After creating new customer from orders/new, I get redirected to customer list, not a new order #17090
2. ✅ After creating new voucher from orders/new, I get redirected to voucher list, not a new order #18146
3. Also when clicking "Details" for a customer, a cart or a previous order, it takes me out of the window, when it should rather open the details with minimal view inside a modal or similar?
--> will be finished in another issue : #18303
4.  ✅ When adding new address for customer, the fields are not prefilled.  Email-field should be locked. #17090
5.  ✅ After adding the address, I am redirected to the addresses list, not the order. #17090
6. After a customer has gotten an address, I am not longer able to create a backoffice order for that customer. The button is gone in favour of a "Edit"-button

PS 1.7.7.x

**Comment by matks** [comment]
I added bug numbers

Bugs 1, 4 and 5 will be fixed by https://github.com/PrestaShop/PrestaShop/pull/17090

**Comment by khouloudbelguith** [comment]
Hi @onklmaps,

>After creating new customer from orders/new, I get redirected to the customer list, not a new order

Same issue with the branch 1.7.7.x
https://drive.google.com/file/d/1JqAJP2gZC5yYdHvaaCJDwGP4oHOv0ENv/view
In progress (https://github.com/PrestaShop/PrestaShop/pull/17090)
>After creating new voucher from orders/new, I get redirected to voucher list, not a new order

Will be fixed later

>After creating new voucher from orders/new, I get redirected to voucher list, not a new order

Blocked by this issue: https://github.com/PrestaShop/PrestaShop/issues/17226
>Also when clicking "Details" for a customer, a cart or a previous order, it takes me out of the window,

All modals are in progress

>When adding new address for customer, the fields are not prefilled. Email-field should be locked.
>After a customer has gotten an address, I am not longer able to create a backoffice order for that customer. The button is gone in favour of a "Edit"-button

Blocked by this issue: https://github.com/PrestaShop/PrestaShop/issues/17226

**Comment by Progi1984** [comment]
Bug 2 is fixed by #18146

**Comment by zuk3975** [comment]
> 6.After a customer has gotten an address, I am not longer able to create a backoffice order for that customer. The button is gone in favour of a "Edit"-button

This seems to be not fixed yet?

**Comment by matks** [comment]
> > 6.After a customer has gotten an address, I am not longer able to create a backoffice order for that customer. The button is gone in favour of a "Edit"-button
> 
> This seems to be not fixed yet?

I was able to create an order by following the scenario
1. Open "Create an order" page
2. Add a new customer
3. All page opens
4. Add an address
5. Choose a product, add it to cart
6. Complete order

So it seems to be fixed
